src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -468,3 +468,21 @@ class LineRunFieldName:
 
 
 TRACE_LIST_DEFAULT_LIMIT = 1000
+
+
+class IdentityConfigurationType(str, Enum):
+    """Enum for identity in YAML."""
+
+    MANAGED = "Managed"
```

Comment:
Do service recognize both Managed & ManagedIdentity?

Reply:
Offline synced, we'll only pass resource id to service